requirements.txt

```diff
@@ -1 +1,2 @@
--e git+https://github.com/poldracklab/niworkflows.git@1c00a2490b7c5c9fd2c5d9afa7b6861d4ccecd08#egg=niworkflows
+-e git+https://github.com/effigies/nipype.git@a5fe6bb#egg=nipype
+-e git+https://github.com/effigies/niworkflows.git@7051b39#egg=niworkflows
```

Comment:
Oh, right. That's still unresolved. See @oesteban's comments on poldracklab/niworkflows#117, where I started to do that. I'm still unsure exactly how to proceed here.